sql/moz-fx-data-shared-prod/firefox_desktop_derived/baseline_active_users_aggregates_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Baseline Active Users Aggregates
+description: |-
+  Builds active_users_aggregates using the baseline ping.
+owners:
+- kwindau@mozilla.com
+- kik@mozilla.com
+labels:
+  incremental: true
+  table_type: aggregate
+scheduling:
+  dag_name: bqetl_analytics_aggregations
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
+    expiration_days: null
+  range_partitioning: null
+  clustering:
+    fields:
+    - app_name
```

Comment:
@irrationalagent I guess this clustering makes sense? Usually from high level we'd filter on app_name and then a lot of our segmentation would happen on channel and country level, right?

Reply:
this seems to mirror the current clustering for AUA, and I can't think of a great reason for changing it